tests/conftest.py

```diff
@@ -81,70 +80,25 @@ def fixture_localhost(aiida_localhost):
     return localhost
 
 
-@pytest.fixture(scope="function")
-def janus_code(aiida_local_code_factory):
-    """
-    Fixture to get the janus code.
-
-    Parameters
-    ----------
-    aiida_local_code_factory : fixture
-        A fixture providing a factory for creating local codes.
-
-    Returns
-    -------
-    `Code`
-        The janus code instance.
+@pytest.fixture(autouse=True)
+def janus_code(aiida_code_installed, aiida_profile):
     """
-    janus_path = shutil.which("janus") or os.environ.get("JANUS_PATH")
-    return aiida_local_code_factory(executable=janus_path, entry_point="mlip.sp")
+    Function-scoped fixture to get the janus Code.
```

Comment:
```suggestion
    Function-scoped fixture to get the janus Code.
```
I'm not against it as such, but why does it make sense to refresh this for every function?